resolve-prebuilds.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path')
+const b4a = require('b4a')
+const unixResolve = require('unix-path-resolve')
+
+module.exports = async function resolvePrebuilds (drive, basedir) {
+  const candidates = getCandidates(basedir)
+  while (candidates.length) {
+    const candidate = candidates.pop()
+    const pkg = await readPackage(drive, candidate)
+    if (pkg) {
+      const name = pkg.name
+      const version = pkg.version
+
+      const prebuildCandidates = [
+        `./prebuilds/${process.platform}-${process.arch}/${name}@${version}.bare`,
+        `./prebuilds/${process.platform}-${process.arch}/${name}@${version}.node`,
+        `./prebuilds/${process.platform}-${process.arch}/${name}.bare`,
+        `./prebuilds/${process.platform}-${process.arch}/${name}.node`
+      ]
```

Comment:
The order is significant and should match https://github.com/holepunchto/bare/blob/deee5b6552be2e25d393fb5d81aee0384d09cfaa/src/addon.js#L165-L170. @mafintosh Come to think of it, should we favor the versioned one over the unversioned one? That is, change the order to:

```js
[
  `${name}@${version}.bare`,
  `${name}.bare`,
  `${name}@${version}.node`,
  `${name}.node`
]
```

Reply:
done